web/json_options.go

```diff
@@ -75,6 +75,7 @@ func defaultJSONOptions() *jsonOptions {
 			{"01-02-2006", false},
 			{"01/02/2006", false},
 			{"01/02/06", false},
+			{"sgnl-unix-ts", true},
```

Comment:
- UNIX timestamp is based on UTC. It does not have timezone info.
- UNIX timestamp is commonly expressed in nanoseconds but systems could use a lower precision like microseconds. Let's be explicit about the precision SGNL supports.
```suggestion
			{"sgnl-unix-ns-timestamp", false},
```